### PR TITLE
[Readme] Add snap and test report status

### PR DESCRIPTION
**Pull request by talregev**
Add snap and test report status.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
...
 | [![Build Status](https://github.com/BOINC/boinc/workflows/Linux/badge.svg)](https://github.com/BOINC/boinc/actions?query=workflow:Linux+branch:master) | [![Coverity Scan Build Status](https://scan.coverity.com/projects/4226/badge.svg)](https://scan.coverity.com/projects/boinc-boinc) |
 | [![Build Status](https://github.com/BOINC/boinc/workflows/Linux-MinGW/badge.svg)](https://github.com/BOINC/boinc/actions?query=workflow:Linux-MinGW+branch:master) | [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/BOINC/boinc/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/BOINC/boinc/?branch=master) |
+| [![Build Status](https://github.com/BOINC/boinc/actions/workflows/snap.yml/badge.svg?branch=master)](https://github.com/BOINC/boinc/actions/workflows/snap.yml) | [![Test Report](https://github.com/BOINC/boinc/actions/workflows/test-report.yml/badge.svg?branch=master)](https://github.com/BOINC/boinc/actions/workflows/test-report.yml)|
```
Comment (AenBleidd) [inline README.md:8]: Please remove `?branch=master` to let GitHub use the default branch

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (talregev) [reply README.md:8]: ok. All the other is using the master branch. So we should fix it for all.

Comment (talregev) [reply README.md]: done.

**Review by AenBleidd — APPROVED** [review]
LGTM. Thank you for the fix.

**Pull request merged**